src/build.props

```diff
@@ -12,6 +12,9 @@
     <VersionPrefix>2.1.0</VersionPrefix>
     <VersionSuffix></VersionSuffix>
     <PackageVersionSuffix>-rtm.0</PackageVersionSuffix>
+    <SarifVersion>2.1.0</SarifVersion>
+    <SchemaVersion>2.1.0-rtm.0</SchemaVersion>
+    <SdkVersion>2.1.0</SdkVersion>
```

Comment:
this is a wip PR to get verbal approval on the approach i am taking here. I have introduced these three variables each for the appropriate purpose of defining:

- SarifVersion - shall remain "2.1.0" to correlate with the spec version.
- SchemaVersion - shall correlate with the schema version on json schema store. current value is [http://json.schemastore.org/sarif-2.1.0-rtm.0](http://json.schemastore.org/sarif-2.1.0-rtm.0)
- SdkVersion - current Semantic version for the SDK. Must be updated before every nuget drop.

Reply:
Add comments to this file that explain the purpose of all these things.

---
In reply to: [282281303](https://github.com/microsoft/sarif-sdk/pull/1484#discussion_r282281303) [](ancestors = 282281303)